lms/djangoapps/verify_student/signals.py

```diff
@@ -28,7 +33,23 @@ def _listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable
             VerificationDeadline.set_deadline(course_key, course.end)
 
 
-@receiver(USER_RETIRE_LMS_CRITICAL)
+@dispatcher.receiver(USER_RETIRE_LMS_CRITICAL)
 def _listen_for_lms_retire(sender, **kwargs):  # pylint: disable=unused-argument
     user = kwargs.get('user')
     SoftwareSecurePhotoVerification.retire_user(user.id)
+
+
+@dispatcher.receiver(post_save, sender=SoftwareSecurePhotoVerification)
+def send_idv_update(sender, instance, **kwargs):  # pylint: disable=unused-argument
+    """
+    Catches the post save signal from the SoftwareSecurePhotoVerification model, and emits
+    another signal with limited information from the model.
```

Comment:
NIT: could be worth pointing out we're doing this so other apps don't need to import the entire model